nixops/backends/__init__.py

```diff
@@ -439,7 +449,8 @@ def copy_closure_to(self, path):
         # Any remaining paths are copied from the local machine.
         env = dict(os.environ)
         env["NIX_SSHOPTS"] = " ".join(
-            ssh._get_flags() + ssh.get_master(user=self.ssh_user).opts
+            ssh._get_flags()
+            + ssh.get_master(user=self.ssh_user, forward_agent=False).opts
```

Comment:
I'm a bit curious about the passing of `forward_agent` too to get_master, etc. We should be careful that plugins have an easy interface to do the thing they want, instead of needing to focus on the details of nixops internals I think?

Reply:
Let's sort that out in a future refactor of the `SSH`/`SSHMaster` classes.